mass/core/files.py

```diff
@@ -251,14 +251,18 @@ def _parse_header(self):
         # Read the new (Feb 2024) subframe information. If missing, assume the old TDM values pertain
         # (so # of rows -> subframe divisions, and row # -> subframe offset), unless source is Abaco,
         # in which case use 64 subframe divisions and offset of 0.
-        default_divisions = self.number_of_rows
         default_offset = self.row_number
+        default_divisions = self.number_of_rows
```

Comment:
I believe this is the only place we should have a fix, and it should have a comment explaining why it works the way it does, and probably print a warning about what is wrong that is making it need the fix. Perhaps "warning: malformed ljh filed with number_of_rows = 0 detected, doing x instead".

Reply:
Okay, I think this basically your request, with minimal other changes.
But we DO have to accommodate older data files that lack this new attribute in the LJH header.